commcare_connect/workflow/templates/mbw_monitoring/views.py

```
@@ -0,0 +1,869 @@
+            try:
+                from commcare_connect.tasks.data_access import TaskDataAccess
+
+                data_access = TaskDataAccess(user=request.user, request=request)
+                all_tasks = data_access.get_tasks()
+                data_access.close()
+                closed_statuses = {"closed", "resolved"}
+                open_task_usernames = sorted(set(
+                    t.username.lower() for t in all_tasks
+                    if t.username and t.status not in closed_statuses
+                ))
+            except Exception as e:
```

Ruff (0.15.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)